docs/api/covidcast-signals/fb-survey.md

```diff
@@ -96,7 +96,7 @@ survey data, we estimate the percentage of people (age 18 or older) who have a
 COVID-like illness, or influenza-like illness, in a given location, on a given
 day.
 
-Signals beginning `raw_w` or `smoothed_w` are [adjusted using survey weights
+Signals beginning `smoothed_w` are [adjusted using survey weights
```

Comment:
```suggestion
Signals beginning `raw_w` or `smoothed_w` are [adjusted using survey weights
```

We should leave `raw_w` here to be clear that they are also weighted estimates.

Reply:
ah yes, overzealous find-and-replace